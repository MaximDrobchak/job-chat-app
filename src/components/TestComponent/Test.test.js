import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import Index from './index.jsx';
describe('Index', () => {
	it('отрисовывает без ошибки', () => {
		const div = document.createElement('div');
		ReactDOM.render(<Index />, div);
		ReactDOM.unmountComponentAtNode(div);
	});
	test('есть корректный снимок', () => {
		const component = renderer.create(<Index />);
		const tree = component.toJSON();
		expect(tree).toMatchSnapshot();
	});
});
