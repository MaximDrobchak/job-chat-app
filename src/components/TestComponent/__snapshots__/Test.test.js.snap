// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Index есть корректный снимок 1`] = `
<div>
  <h1>
    Hello, world!
  </h1>
  <h2>
    It is 
    22:35:25
    .
  </h2>
</div>
`;
